### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by vikashyap99**
Change in background-color and div-color.

#### Checklist

- [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [ ] There is a corresponding issue for this pull request.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [ ] The unit tests pass locally with my changes (provide a screenshot or link for test) 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

-
-
-

Fixes #

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -36,6 +39,13 @@
   <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
   <link href="https://netdna.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.css" rel="stylesheet">
   <title>Meilix ISO Generator</title>
+  <style>
+  
+    body{
+        background: skyblue; font-family: verdana; color:#fff; padding:10px;
```
Comment (tabesin) [inline templates/index.html]: Please no inline CSS

File: templates/index.html [hunk 2]
```diff
@@ -57,9 +67,11 @@
   <form method="POST" enctype="multipart/form-data">
 
 <div class="custom-navbar">
-    <a style="text-decoration: none;" href='.' class="custom-navtitle">
-      <strong>Meilix Generator</strong>
+  <p><font size="6px"></p>
```
Comment (tabesin) [inline templates/index.html]: We use CSS. not inline font tags

**Comment by sarnava1** [comment]
@tabesin why are we changing the css? The present bg color and other styles goes well with other settings? Was there any need for any UI changes now?

**Comment by tabesin** [comment]
@sarnava1 that is another issue.

**Comment by sarnava1** [comment]
Ohh! Okay @tabesin !! Can you tell me the issue no @tabesin?

**Comment by tabesin** [comment]
@sarnava1 The other concern is that this is a pull request without an "issue" in the tracker. The author also does not tell us why he makes these changes. All this could be provided. A related issue could be filed.

However, the formal reason to reject the PR - as it is - is the lack of compliance with MVC principles. Changes have to me made in the CSS (when you agree with the more fundamental notion you raised if it is necessary). Furthermore if we make a change that concerns visuals to provide before-after visuals screenshots that demonstrate the effects of the screenshots.

The author of this PR is new to the project, so it does not make sense to demotivate him where he could rectify the PR and also file a related issue. I think for trivial changes we do not always have to be dogmatic about related issues in the tracker, this is a educative principle, a good practice.

**Comment by sarnava1** [comment]
@tabesin yes you are perfect with your explanations. @vikashyap99 please do go through best practises and also see the previous comment of @tabesin for good contributions.

**Comment by tabesin** [comment]
For the Fossasia contribution guidelines please read and follow:
https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia/
I know that it is often inconvenient to adapt to specific rules that define the development workflow and slow down your productivity at first sight but adhering to these principles is very useful for making our development productive and maintainable and you will see the advantages yourself when applying to the best practices. Esp. when you are one day on the other side and have to review code contributions. Also note that in large scale multi user development environments our approach is pretty agile and useful for ISO 90000/ITIL compliance and it is used consistently for all different development projects under our umbrella. I know that at times we have to deviate for practical reasons. @meets2tarun may tell you the story how we transitioned from unstructured contributions to Meilix to a more Fossasia standards compliant development process.

**Comment by vikashyap99** [comment]
Thanks everyone for your guidence. Next i will try to PR according to requirement of a project

**Comment by meets2tarun** [comment]
@vikashyap99 
@tabesin has already guided you, go through the blog and follow on.

Thanks

**Pull request closed without merging**